eversql/README.md

```diff
@@ -0,0 +1,73 @@
+# EverSQL - Your Database, Just Faster™
+
+## Overview
+- Looking to speed up slow SQL queries?
+- Reached 100% CPU on your database, and you are unsure how to fix it?
+- Database storage is out of space and looking for redundant indexes that you can delete?
+**
+
+[EverSQL][1] is the fastest way to automatically speed up your database and optimize SQL queries.
+80,000+ developers, DBAs, and DevOps engineers use EverSQL for automatic SQL tuning and indexing.
+Our customers report their queries are 5X faster on average, just minutes after getting started.
+Save your team 35 weekly hours on average by optimizing your SQL queries online for free.
+
+EverSQL is 100% non-intrusive, and doesn't access any of your databases' sensitive data.
+
+![snapshot][8]
+
+### Usage
+
+The EverSQL integration has two options:
+
+- A manual method, where you copy your queries from Datadog dashboard and paste them to EverSQL’s [optimize a query][2] action.
+- An automatic method, where you add the EverSQL Sensor to your [RDS][3] or your [on-prem database][4].
+
+### Supported Databases: 
+MySQL, PostgreSQL, Aurora,Cloud SQL, Azure DB, Percona, MariaDB.
+
+## Setup
+
+### Configuration
+In order to use the manual workflow, just start at [EverSQL][2].
```

Comment:
Are any specific steps needed to get your Datadog account set up in EverSQL (i.e. entering an API key)?

Reply:
At the current step, datadog users copy the data (sql queries) from datadog into EverSQL and can get them optimized directly, so no need for API keys at the moment.

Reply:
I see - it might be helpful to lay out those specifics here, such as where one would find their queries within Datadog, or the types of queries they should be pasting into EverSQL